Bot/cogs/remindme.py

```diff
@@ -54,6 +54,21 @@ def cog_unload(self):
         await self.redis.hdel("{}:Remindme:channel".format(guild), x)
         await self.redis.hdel("{}:Remindme:time".format(guild), x)
 
+    @commands.command(hidden=True,pass_context=True)
+    async def remindtime(self,ctx,get_time,*,message=""):     
+        time = get_time.split(":")
+        if not time[0].isdigit():
+            return await self.bot.say(ctx,content = "You enter the format wrong! It should be look like this {}remindtime hh:mm:ss message".format(ctx.prefix))
+        if len(time) == 2:
+          time[2] = '0'
+        if len(time) == 1:
+          time[1] = '0'
+        delta_time = datetime.utcnow().replace(hour=int(time[0]),minute=int(time[1]),second=int(time[2])) - datetime.utcnow()
+        if delta_time < 0:
+            delta_time += datetime.timedelta(days=1)
+            
+        await self.remindme(self,ctx,str(delta_time),*,message="")
```

Comment:
This not how you call object function, plus Command object are not callable.
Good try tho.